content/en/docs/instrumentation/net/automatic.md

```diff
@@ -100,3 +100,9 @@ You can also find more instrumentations available in the [opentelemetry registry
 
 After you have observability generated automatically with instrumentation libraries, you may want to add
 [manual instrumentation]({{< relref "manual" >}}) to collect custom telemetry data.
+
```

Comment:
- See #1042

Reply:
We have a separate issue open already to track the concern, so I'll mark this as resolved.